### PR TITLE
ci: move to fleet-ci

### PR DESCRIPTION
**Pull request by v1v**
## What is the problem this PR solves?

Use `fleet-ci`  and the folder name `fleet-server` to host all the specific Jenkins Jobs related to the `fleet-server`.

NOTE: this PR will fail in `beats-ci` 

## Why

In order to distribute the load and reduce a single point of failure

## Actions

- [ ] Agree when to merge
- [ ] Create webhook to point to `fleet-ci`

### CODE REVIEW

**Review by cachedout — APPROVED** [review]

**Comment by ph** [comment]
@v1v Is this still valid?

**Comment by v1v** [comment]
> @v1v Is this still valid?

Yes, I didn't move forward as we wanted to stress the new CI controller with some k8s agents, but we are still blocked by this stress testing, so I'd rather prefer to wait a bit more to let us work on the stress testing without affecting anyone, if you don't mind 🙏

**Comment by cachedout** [comment]
@v1v Are you still blocked on this?

**Comment by v1v** [comment]
This is ready to be reviewed, we won't use `fleet-ci` for anything else but running the CI as it's production ready

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://beats-ci.elastic.co/job/Ingest-manager/job/fleet-server/job/main/badge/icon)](https://beats-ci.elastic.co/job/Ingest-manager/job/fleet-server/job/main/)
+[![Build Status](https://fleet-ci.elastic.co/job/Ingest-manager/job/fleet-server/job/main/badge/icon)](https://fleet-ci.elastic.co/job/Ingest-manager/job/fleet-server/job/main/)
```
Comment (v1v) [inline README.md:1]:
```suggestion
[![Build Status](https://fleet-ci.elastic.co/job/fleet-server/job/fleet-server-mbp/job/main/badge/icon)](https://fleet-ci.elastic.co/job/Ingest-manager/job/fleet-server/job/main/)
```

**Pull request merged**